logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -61,7 +61,6 @@
     private final int maxUnread;
     private final int checkpointMaxAcks;
     private final int checkpointMaxWrites;
-    private final int checkpointMaxInterval;
```

Comment:
I'm guessing that this was intended to write a checkpoint after some amount of time, regardless of the number of acks/writes, but was never implemented?